parlai/core/logs.py

```diff
@@ -9,25 +9,24 @@
 This file provides interface to log any metrics in tensorboard, could be
 extended to any other tool like visdom.
 
-If you use tensorboard logging, all event folders will be stored in
-``PARLAI_DATA/tensorboard`` folder. In order to Open it with TB, launch
-tensorboard as:
-
 .. code-block: none
 
    tensorboard --logdir <PARLAI_DATA/tensorboard> --port 8888.
 """
 
-# TODO: update this with pytorch 1.1's tensorboard API
```

Comment:
This is intentional. The pytorch 1.1/1.2 API is incomplete and tensorboardX works better.